src/js/components/homepage/AwardSearch/AwardSearch.jsx

```diff
@@ -53,6 +61,59 @@ const AwardSearch = () => {
         return () => window.removeEventListener('resize', handleResize);
     }, []);
 
+
+    const getSelectedTab = (tab, rankType) => {
+        const filterValue = {
+            filters: {
+                ...defaultFilters
+            },
+            version: "2020-06-01"
```

Comment:
This could be confusion between we're now hardcoding the version in multiple locations.   Adding the data to the GlobalConstants file would prevent confusion later on